### PR TITLE
Add Hate Speech Dataset in Filipino

### PR DESCRIPTION
**Pull request by anaerobeth**
This PR adds the Hate Speech Dataset, a text classification dataset in Filipino, consisting 10k tweets (training set) that are labeled as hate speech or non-hate speech. Released with 4,232 validation and 4,232 testing samples. Collected during the 2016 Philippine Presidential Elections.

Link to the paper: https://pcj.csp.org.ph/index.php/pcj/issue/download/29/PCJ%20V14%20N1%20pp1-14%202019

Link to the dataset/repo: https://github.com/jcblaisecruz02/Filipino-Text-Benchmarks

### CODE REVIEW

**Review by lhoestq — APPROVED** [review]
Nice ! thanks for adding this one

File: datasets/hate_speech_filipino/README.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+## Dataset Structure
+
+### Data Instances
+
+Sample data:
+{
+  "text": "Taas ni Mar Roxas ah. KULTONG DILAW NGA NAMAN",
+  "label": 1
+}
```
Comment (lhoestq) [inline datasets/hate_speech_filipino/README.md:74]:
```suggestion
```
{
  "text": "Taas ni Mar Roxas ah. KULTONG DILAW NGA NAMAN",
  "label": 1
}
```
```

File: datasets/hate_speech_filipino/hate_speech_filipino.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    VERSION = datasets.Version("1.0.0")
+
+    def _info(self):
+        # Labels: 0="Non-hate Speech", 1="Hate Speech"
+        features = datasets.Features(
+            {
+                "text": datasets.Value("string"),
+                "label": datasets.features.ClassLabel(names=["0", "1"])
+            }
+        )
```
Comment (lhoestq) [inline datasets/hate_speech_filipino/hate_speech_filipino.py:59]:
```suggestion
        features = datasets.Features(
            {"text": datasets.Value("string"), "label": datasets.features.ClassLabel(names=["0", "1"])}
        )
```
code formatting
